src/compiler/commandLineParser.ts

```diff
@@ -320,6 +320,11 @@ namespace ts {
             name: "allowSyntheticDefaultImports",
             type: "boolean",
             description: Diagnostics.Allow_default_imports_from_modules_with_no_default_export_This_does_not_affect_code_emit_just_typechecking
+        },
+        {
+            name: "emitModulesInLooseMode",

```

Comment:
Coming from babel, ["loose mode"](https://babeljs.algolia.com/docs/advanced/loose/) means a set of operations are allowed to be performed not-to-spec (so that some corner cases don't function as expected) - TS output (by comparison) is always loose mode.

What about `omitUseStrictPrologue`?